src/resources/parser/scene.js

```diff
@@ -49,7 +49,7 @@ class SceneParser {
     }
 
     _createEntity(data, compressed) {
-        const entity = new Entity();
+        const entity = new Entity(this._app);
```

Comment:
ideally we call constructor correctly, which is `    constructor(name, app)`
the Entity constructor class handles app first, but parent class GraphNode does not seem to and would attach application as a name I think.

Reply:
Fixed